loadflow-validation/src/main/java/eu/itesla_project/loadflow/validation/CheckFlowsConfig.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.loadflow.validation;
+
+import java.util.Objects;
+
+import eu.itesla_project.commons.config.ComponentDefaultConfig;
+import eu.itesla_project.commons.config.ModuleConfig;
+import eu.itesla_project.commons.config.PlatformConfig;
+import eu.itesla_project.commons.io.table.CsvTableFormatterFactory;
+import eu.itesla_project.commons.io.table.TableFormatterFactory;
+import eu.itesla_project.loadflow.api.LoadFlowFactory;
+
+/**
+ *
+ * @author Massimo Ferraro <massimo.ferraro@techrain.it>
+ */
+public class CheckFlowsConfig {
+    
+    public static final float THRESHOLD_DEFAULT = 0.0f;
+    public static final boolean VERBOSE_DEFAULT = false;
+    public static final Class<? extends TableFormatterFactory> TABLE_FORMATTER_FACTORY_DEFAULT =  CsvTableFormatterFactory.class;
```

Comment:
Should be private?
Remove extra space after "TABLE_FORMATTER_FACTORY_DEFAULT =  CsvTableFormatterFactory.class;"